extensions/amp-story/0.1/audio.js

```diff
@@ -0,0 +1,429 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {dev} from '../../../src/log';
+
+/**
+ * The volume to which audio should be reduced when other audio sources are
+ * played.
+ * @const {number}
+ */
+const REDUCED_VOLUME = 0.5;
+
+/**
+ * @const {number}
+ */
+const VOLUME_CHANGE_DURATION_MS = 500;
+
+/**
+ * @const {!function(number): number}
+ */
+const VOLUME_EASING_FN = input => input;
+
+/**
+ * @const {string}
+ */
+const PLAYABLE_ID_PREFIX = 'i-amphtml-playable-';
```

Comment:
nit: maybe `i-amphtml-playable-audio-` to account for other playable types later given `nextId_` is scoped to `audio.js` only?

Reply:
Done